### DISCUSSION TITLE
`kit` not defined in original scripts like `Convert Colors`

### DISCUSSION THREAD

**Discussion by AlanBreck** [johnlindquist/kit#506] [Errors] [answered]
I'm using `v1.3.0-beta.66`

<img width="339" alt="Screen Shot 2021-11-04 at 10 06 16 PM" src="https://user-images.githubusercontent.com/1199820/140446230-bc69de39-c200-4e10-902a-ed76266c1d65.png">

**Comment by johnlindquist** [comment, marked as answer]
You can safely remove any line of code that has "await kit" on it. Those methods that were originally imported off of kit are now in the global scope.

**Reply by AlanBreck** [reply]
Awesome! Thank you for your quick reply.
